themer.js

```diff
@@ -0,0 +1,17 @@
+// document must have this element
+const themeSwitch = document.getElementById('theme-switch');
+
+document.addEventListener('DOMContentLoaded', function(e) {
+    themeSwitch.checked = localStorage.getItem(themeSwitch.id);
+    setTheme();
+    themeSwitch.addEventListener('change', setTheme);
+});
+
+
+// saves theme to storage and set's body attr
+const setTheme = () => {
+    localStorage.setItem(themeSwitch.id, themeSwitch.checked ? true : "")
```

Comment:
nit: missing semi-colon. Don't really care but we may as well be consistent.

Also might be able to do this instead (again, I don't really care one way or the other, just a thought):

```javascript
localStorage.setItem(themeSwitch.id, themeSwitch.checked || "");
```